barretenberg/cpp/pil/avm/avm_mini.pil

```diff
@@ -21,8 +21,12 @@ namespace avmMini(256);
     // DIV
     pol commit sel_op_div;
 
-    // Error boolean flag pertaining to an operation
-    pol commit op_err;
+    // Instruction memory tag (0: uninitialized, 1: u8, 2: u16, 3: u32, 4: u64, 5: u128, 6:field)
```

Comment:
in what instances do we want uninitialized?

Reply:
Not sure if Noir relies on this but I understood that we would interpret uninitialized memory in this way. I have mentioned this to @dbanks12 and we should align with Noir team.The final decision seems to be pending on whether we should be strict (disallow reading uninitialized memory) or not.